app/organisation/rest.py

```diff
@@ -39,6 +39,8 @@
 from app.organisation.organisation_schema import (
     post_create_organisation_schema,
     post_link_service_to_organisation_schema,
+    post_notify_org_member_about_next_steps_of_go_live_request,
+    post_notify_service_member_of_rejected_request_to_go_live,
```

Comment:
[naming nitpick, optional]  change to `post_notify_service_member_of_rejected_go_live_request` so both functions have `go_live_request` wording.